src/RoboFileBase.php

```diff
@@ -420,13 +422,18 @@ public function digipolisInstallDrupal8(
     ) {
         $this->readProperties();
         $app_root = $this->getConfig()->get('digipolis.root.web', false);
-        $site_path = 'sites/default';
+        $site_path = $app_root . '/sites/default';
 
-        if (is_file($app_root . '/sites/default/settings.php')) {
-            include $app_root . '/sites/default/settings.php';
+        if (is_file($site_path . '/settings.php')) {
+            chmod($site_path . '/settings.php', 664);
+            include $site_path . '/settings.php';
         }
-        elseif (is_file($app_root . '/sites/default/settings.local.php')) {
-            include $app_root . '/sites/default/settings.local.php';
+        elseif (is_file($site_path . '/settings.local.php')) {
```

Comment:
When is there a situation where there is no settings.php but there is a settings.local.php?

Reply:
Don't know, but it's supported ;-)